src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css

```diff
@@ -1,9 +1,19 @@
-﻿@import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
+@import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
 
 html, body {
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
 }
 
+a, .btn-link {
+    color: #0366d6;
+}
+
+.btn-primary {
```

Comment:
@adrianwright109 could you explain why these were changed? This seems unrelated to the linked issue. We'd prefer an absolute minimal PR that fixes the addressed issue.

Reply:
I assumed all 5 `site.css` should be the same so added the missing sections so that they are now all the same.

Reply:
@pranavkm Having reviewed this now, I agree it's good to make these changes to bring all the CSS into consistency. If you're OK with that, could you withdraw your "request changes" so I can merge this?